releng/_go_versions.sh

```diff
@@ -1,5 +1,5 @@
 # These are the current and "next" Go versions used to build influxdb.
 # This file is meant to be sourced from other scripts.
 
-export GO_CURRENT_VERSION=1.12.7
-export GO_NEXT_VERSION=1.12.7
+export GO_CURRENT_VERSION=1.13.8
+export GO_NEXT_VERSION=1.14
```

Comment:
Let's set both of these to 1.13.8 for now. I'm fuzzy on the history but I think we only set the one ahead for testing.

Reply:
okay.